kadi/commands/core.py

```diff
@@ -1080,14 +1080,14 @@ def get_par_idx_update_pars_dict(pars_dict, cmd, params=None, rev_pars_dict=None
     return par_idx
 
 
-def ska_load_dir(load_name):
-    root = Path(os.environ["SKA"]) / "data" / "mpcrit1" / "mplogs"
-    year = load_name[5:7]
-    if year == "99":
-        year = 1999
-    else:
-        year = 2000 + int(year)
-    load_rev = load_name[-1].lower()
-    load_dir = load_name[:-1]
-    load_dir = root / str(year) / load_dir / f"ofls{load_rev}"
-    return load_dir
+def ska_load_dir(load_name: str) -> Path:
+    """Return a load directory path in $SKA/data from a load name like "DEC1123B"."""
+
+    warnings.warn(
+        "ska_load_dir() is deprecated. Use parse_cm.paths.load_dir_from_load_name()",
+        np.VisibleDeprecationWarning,
```

Comment:
What is the issue with using `FutureWarning` here? (as you suggest) It would seem to me that this is exactly what it is meant for, but perhaps I'm missing something.

Strictly speaking, maybe the intended use is to create our own deprecation warning class and make it inherit from `FutureWarning`, though I am not proposing to do that.

Reply:
I guess I like that the numpy one has the word Deprecation in it, but I see the point and have changed this.